src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Orchard.ContentManagement;
+using Orchard.Data;
+using Orchard.Environment.Extensions;
+using Orchard.Localization;
+using Orchard.Logging;
+using Orchard.Roles.Models;
+using Orchard.Roles.Services;
+using Orchard.Workflows.Models;
+using Orchard.Workflows.Services;
+
+namespace Orchard.Roles.Activities {
+    [OrchardFeature("Orchard.Roles.Workflows")]
+    public class UnssignRoleActivity : Task {
+        private readonly IWorkContextAccessor _workContextAccessor;
+        private readonly IRepository<UserRolesPartRecord> _repository;
+        private readonly IRoleService _roleService;
+
+        public UnssignRoleActivity(
+            IWorkContextAccessor workContextAccessor,
+            IRepository<UserRolesPartRecord> repository,
+            IRoleService roleService) {
+            _workContextAccessor = workContextAccessor;
+            _repository = repository;
+            _roleService = roleService;
+            T = NullLocalizer.Instance;
+            Logger = NullLogger.Instance;
+        }
+
+        public Localizer T { get; set; }
+        public ILogger Logger { get; set; }
+
+        public override string Name
+        {
```

Comment:
I do not have any other extension on my VS
this is what I have in Rebracer.xml under "src". In particular         **NewLines_Braces_Property 1** Is that correct?

```
        <PropertyValue name="NewLines_AnonymousTypeInitializer_EachMember">1</PropertyValue>
        <PropertyValue name="NewLines_Braces_Accessor">1</PropertyValue>
        <PropertyValue name="NewLines_Braces_AnonymousMethod">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_AnonymousTypeInitializer">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_ArrayInitializer">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_CollectionInitializer">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_ControlFlow">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_LambdaExpressionBody">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_Method">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_ObjectInitializer">0</PropertyValue>

        <PropertyValue name="NewLines_Braces_Property">1</PropertyValue>

        <PropertyValue name="NewLines_Braces_Type">0</PropertyValue>
        <PropertyValue name="NewLines_Keywords_Catch">1</PropertyValue>
        <PropertyValue name="NewLines_Keywords_Else">1</PropertyValue>
        <PropertyValue name="NewLines_Keywords_Finally">1</PropertyValue>
        <PropertyValue name="NewLines_ObjectInitializer_EachMember">1</PropertyValue>
        <PropertyValue name="NewLines_QueryExpression_EachClause">1</PropertyValue>

```